src/pages/transactiondetail/index.tsx

```diff
@@ -0,0 +1,558 @@
+            </div>
+            <div className="flex flex-col">
+              <NumericFormat
+                displayType="text"
+                className="text-white-50 whitespace-normal lg:w-[247px] md:w-[137px] w-[132px] lg:text-left md:text-left text-right"
+                thousandSeparator
+                value={useUnitSuffix(gasUsed)}
+              />
+              <NumericFormat
+                displayType="text"
+                className="text-white-700 text-xs mt-1 lg:w-[247px] md:w-[137px] w-[132px] lg:text-left md:text-left text-right"
+                thousandSeparator
+                value={BigNumber(gasUsedPercentage).toFixed(2)}
+                suffix="%"
+              />
+            </div>
+          </div>
+        </div>
+        {type === "tokenized" ? (
+          <div className="flex flex-col gap-y-6 lg:w-1/2 md:w-1/2">
+            <div className="text-white-50">{fixedTitle[11]}</div>
+            <div className="flex flex-row">
+              <div className="grow">
+                <div className="text-white-700 w-[101px]">{fixedTitle[12]}</div>
+              </div>
+
+              <div
+                className={clsx(
+                  "flex flex-row items-center gap-x-2.5 md:text-left lg:w-[247px] md:w-[137px] w-auto",
+                  { "md:w-[155px] w-auto gap-x-0": copyFromAddress }
+                )}
+              >
+                <div className={clsx({ "mr-[10px]": copyFromAddress })}>
+                  <CopyableText
+                    label={truncateTextFromMiddle(from)}
+                    onTextClick={setCopyFromAddress}
+                    textClicked={copyFromAddress}
+                  />
+                </div>
+                {copyFromAddress ? (
+                  <div>
+                    <GreenTickIcon />
+                  </div>
+                ) : (
+                  <FiCopy className="text-white-50 h-[22px]" />
+                )}
+              </div>
+            </div>
+            <div className="flex flex-row">
+              <div className="text-white-700 w-[101px] grow">
+                {fixedTitle[13]}
+              </div>
+              <div
+                className={clsx(
+                  "flex flex-row items-center gap-x-2.5 md:text-left lg:w-[247px] md:w-[137px] w-auto",
+                  { "md:w-[155px] w-auto gap-x-0": copyToAddress }
+                )}
+              >
+                <div className={clsx({ "mr-[10px]": copyToAddress })}>
+                  <CopyableText
+                    label={truncateTextFromMiddle(to)}
+                    onTextClick={setCopyToAddress}
+                    textClicked={copyToAddress}
+                  />
+                </div>
+                {copyToAddress ? (
+                  <div>
+                    <GreenTickIcon />
+                  </div>
+                ) : (
+                  <FiCopy className="text-white-50 h-[22px]" />
```

Comment:
clicking on copy icon is the copy to clipboard action

Reply:
added copy icon onclick copy to clipboard